### ISSUE TITLE
Error Occurs When Accessing docs.bsky.app

### ISSUE THREAD

**Issue by hg-pyun** [bluesky-social/bsky-docs#157]
Hello,

I encountered an issue when trying to access docs.bsky.app. Here are the details of the problem:

Error Encountered: When attempting to access the website, I receive an error message stating "This page can't be displayed."

URL Attempted: https://www.docs.bsky.app/

Steps to Reproduce:

Open a web browser and go to https://www.docs.bsky.app/
The page fails to load and an error message is displayed.
Browser Used: (Please fill in the browser and version you used. Example: Chrome 114.0.5735.199, Firefox 113.0.2, etc.)

Operating System: (Please fill in your operating system. Example: Windows 10, macOS 13.4, etc.)

I would appreciate it if you could look into this issue. Please let me know if you need any additional information.

Thank you.

**Comment by qazmlp** [comment]
The docs are at https://docs.bsky.app. (Is there a place that links to it with the www. prefix?)

**Comment by bnewbold** [comment]
Yup! We had www.docs.bsky.app until a couple days ago, when we discovered some security / maintenance issues with "www" prefix domain configurations, and removed them when not needed.
